devtools/zenodo/zenodo_data_release.py

```diff
@@ -331,22 +368,20 @@ def get_html_url(self):
 )
 @click.option(
     "--source-dir",
-    type=click.Path(
-        exists=True, file_okay=False, dir_okay=True, readable=True, path_type=Path
-    ),
+    type=str,
     required=True,
-    help="Path to a directory whose contents will be uploaded to Zenodo. Subdirectories are ignored.",
+    help="Path to a directory whose contents will be uploaded to Zenodo. "
+    "Subdirectories are ignored. Can get files from GCS as well - just prefix "
+    "with gs://.",
```

Comment:
Would this also work with `s3://` paths?

I guess there's no real benefit from that though, since we're pulling the data down and then pushing it up rather than copying between remote locations directly. So it'll probably be faster and less flaky to pull from GCS since we're running on teh google.